### PR TITLE
add a nofeatures model-type option

### PR DESCRIPTION
**Pull request by JacobLemieux**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -64,11 +64,11 @@ analyze: FORCE
...
-	python scripts/mutrans.py --backtesting-max-day `seq -s, 150 14 220` --forecast-steps 12
-	python scripts/mutrans.py --backtesting-max-day `seq -s, 220 14 500` --forecast-steps 12
-	python scripts/mutrans.py --backtesting-max-day `seq -s, 500 14 625` --forecast-steps 12
-	python scripts/mutrans.py --backtesting-max-day `seq -s, 626 14 700` --forecast-steps 12
-	python scripts/mutrans.py --backtesting-max-day `seq -s, 710 14 766` --forecast-steps 12
+	python scripts/mutrans.py --backtesting-max-day `seq -s, 150 14 220` --forecast-steps 12 --model-type reparam-localinit-nofeatures
+	python scripts/mutrans.py --backtesting-max-day `seq -s, 220 14 500` --forecast-steps 12 --model-type reparam-localinit-nofeatures
+	python scripts/mutrans.py --backtesting-max-day `seq -s, 500 14 625` --forecast-steps 12 --model-type reparam-localinit-nofeatures
+	python scripts/mutrans.py --backtesting-max-day `seq -s, 626 14 700` --forecast-steps 12 --model-type reparam-localinit-nofeatures
+	python scripts/mutrans.py --backtesting-max-day `seq -s, 710 14 766` --forecast-steps 12 --model-type reparam-localinit-nofeatures
```
Comment (fritzo) [inline Makefile:71]: You might want to create a new makefile target called `backtesting-nofeatures` so that nicks old command still works

**Review by fritzo — COMMENTED** [review]
Looks good except for the Makefile thing

**Review by fritzo — APPROVED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (JacobLemieux) [reply Makefile]: thanks, added backtesting-nofeatures to the makefile

**Pull request merged**